src/editors/containers/VideoEditor/components/VideoSettingsModal/components/DurationWidget/index.scss

```diff
@@ -0,0 +1,5 @@
+@import "@edx/paragon/scss/core/core";
+
+.total-label {
+    border: 1px solid $gray-500;
```

Comment:
@0x29a IIRC, @xitij2000 was working on reducing the usage of SCSS variables, so that they are not needed during compile time. I think you can replace these styles with [Bootstrap css classes](https://getbootstrap.com/docs/4.0/utilities/borders/).

@xitij2000 Is this the right approach?

Reply:
@Cup0fCoffee, sure, I can change that. My intention was to make the outline of the same color as form controls.

Reply:
@0x29a I don't know if the method I suggested is the preferred way. I would wait for @xitij2000 to comment, if you can afford to wait.

> My intention was to make the outline the same color as form controls.

Sorry for the confusion. I think you can use `border-gray-500` class. I'm not sure if it's defined in Bootstrap or edx-paragon, but it seems to be available, and does have the same color as the form controls.

Reply:
>I think you can use border-gray-500 class.

@Cup0fCoffee, cool, this is exactly what I was looking for but didn't find. I'll remove this file and use this class once we have proper mockups.